v4m-container/README.md

```diff
@@ -0,0 +1,102 @@
+# V4M Docker Container
+
+**EXPERIMENTAL**:  Container of the [viya4-monitoring-kubernetes](https://github.com/sassoftware/viya4-monitoring-kubernetes)
+project
+
+## Pre-requisites
+
+- Docker installed on your workstation
+
+## Preparating the Docker Container
+
+### Docker Image
+
+Run the following command to create the `v4m` Docker image using the provided Dockerfile
+
+```bash
+# Move to v4m-container directory
+cd v4m-container
+
+# Build the Docker container
+docker build --no-cache -t v4m .
+```
+
+The Docker image `v4m` will contain Helm, kubectl, and other executables needed to run the v4m deployment.
+
+#### Kubeconfig File
+
+To ensure that your kubeconfig files are available from within the Docker container, you can copy the kubeconfig files to the `./v4m-container/kubeconfig` directory or mount the file or directory during your `docker run` command.
+
+#### (Optional) Custom Deployment Configurations
+
+If you have made changes using a USER_DIR directory, you can copy the contents of this directory to the `./v4m-container/user_dir` directory or mount the file or directory during your `docker run` command.
+
+## Running the Docker Container
+
+### Important Directory Locations in the Docker Container
+
+The main files that you will be working with in the Docker container are in the following locations:
+
+```bash
+# V4M Deployment:
+/opt/v4m/viya4-monitoring-kubernetes/
+
+# kubeconfig Files:
+/opt/v4m/.kube/
+
+# User Directory
+/opt/v4m/user_dir/
+```
+
+### Connecting to the Docker Container
+
+To connect to the Docker container, run the following command:
+
+```bash
+docker run -it v4m
+```
+
+From there, you will be in your UNIX environment.  Before you run any of the V4M scripts, you
+will need to set the `KUBECONFIG` environment variable using the following command:
+
+```bash
+export $KUBECONFIG=/opt/v4m/.kube/<name of kubeconfig file>.conf
+```
+
+Now you can run any script from the `/opt/v4m/viya4-monitoring-kubernetes` directory in the Docker container.
+
+### Running Commands from Outside of the Docker Container
+
+To run commands from outside of the Docker Container, you can run commands similar to the ones below:
+
+```bash
+docker run -it /
```

Comment:
Are the '-it' flags necessary/appropriate when running from outside the container? Are they only needed for interactive and console (SSH-like) access?